tools/oven/src/DomainBaker.cpp

```
@@ -220,7 +222,8 @@ void DomainBaker::addTextureBaker(const QString& property, const QString& url, i
 
         // add this QJsonValueRef to our multi hash so that it can re-write the texture URL
         // to the baked version once the baker is complete
-        _entitiesNeedingRewrite.insert(textureURL, { property, jsonRef });
+        // it doesn't really matter what this key is as long as it's consistent
+        _entitiesNeedingRewrite.insert(textureURL.toDisplayString() + type, { property, jsonRef });
```

Just in case, can we throw in an invalid URL character in here to make sure there's no name collisions?